Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js

```diff
@@ -1,12 +1,12 @@
 const { readdirSync } = require('fs');
 
-const asDirent = { withFileTypes: true };
```

Comment:
This looks more like an abbreviation for "as directory entry". Check if indeed, and use a non-abbreviated name then. Same for `dirent` below.